components/hardware/camera/astraRGBD/README.md

```diff
@@ -71,21 +71,24 @@ After editing the new config file we can run the component:
     ./bin/astraRGBD --Ice.Config=config
 
 ## Final notes
-Just some of the main functions of the rgbd interface have been currently implemented.
-It's under development.
+`astraRGBD` is currently under development and hence only some of the main functions of the `RGBD.idsl` interface have been implemented.
 
 ## Known issues
-The AstraSDK-v__XXX__-Linux/samples$ ./SimpleBodyViewer-SFML sometimes fails silently showing just a black screen.
-You can change the messages shown on on the terminal by the example modifying the lib/astra.toml and set
-the  
-**level = "trace"** or **level = "debug"**  
-If you do so, you can see all the steps done inside the astraSDK when launching the example.
+The command
+```
+./bin/SimpleBodyViewer-SFML
```

Comment:
It's important for the user that it's on the AstraSDK examples. Not in the Robocomp folder structure.

Reply:
okay I will fix that